sdk/core/azure_core_amqp/Cargo.toml

```
@@ -40,6 +40,7 @@ tracing-subscriber = { workspace = true, features = ["env-filter"] }
 default = ["fe2o3_amqp"]
 cplusplus = []
 test = []
+amqp_tracing = ["fe2o3-amqp/tracing"]
```

A few thoughts come to mind. First and foremost - and I've started taking this stance with various `reqwest` options in `azure_core` - I don't think we want to start 1) trying to match parity with dependencies' features, and 2) advertise specific dependencies. At least 2 you did, but because `fe2o3_amqp` is optional, this should be:


```suggestion
amqp_tracing = ["fe2o3-amqp?/tracing"]
```

If we added support for other backends in the future, you could add them to this single feature.

That said, and considering what I said above, wouldn't it be better that someone just take a dependency on `fe2o3-amqp` themselves and add the feature? Resolve 2+ will unify (combine) them. This is the stance I think we want to take with various `reqwest` features apart from a few sane options like gzip and deflate support. Maybe even `rustls` given the issues that's causing for some customers.

/cc @RickWinter

---

And if we go that route, in your crate's `Cargo.toml` either add their `tracing` feature in your `dev-dependencies` for `fe2o3-amqp` (a separate import that adds features - I do this elsewhere and it works fine), or if you just want to do it for this one test, you could try declaring a `[[test]]` table just for this one file and add `required-features = ["fe2o3-amqp/tracing"]` and see if that works; though, I honestly don't know if it will...and kinda doubt it will, but worth a try.